sonar-project.properties

```
@@ -0,0 +1,7 @@
+sonar.projectName=digital-marketplace
+sonar.sources=src/front-end,src/back-end
+sonar.sourceEncoding=UTF-8
+
+sonar.organization=bcgov-sonarcloud
+sonar.projectKey=digital_marketplace
```

```suggestion
sonar.projectKey=bcgov_digital_marketplace
```
In SonarCloud the `projectKey` is prefixed by the github org name and an underscore. See `bcgov_cas-ciip-portal` for [an example](https://sonarcloud.io/project/overview?id=bcgov_cas-ciip-portal).